redbeams/src/main/java/com/sequenceiq/redbeams/flow/redbeams/upgrade/RedbeamsUpgradeFlowConfig.java

```diff
@@ -30,10 +26,8 @@ public class RedbeamsUpgradeFlowConfig extends AbstractFlowConfiguration<Redbeam
 
     private static final List<Transition<RedbeamsUpgradeState, RedbeamsUpgradeEvent>> TRANSITIONS =
             new Transition.Builder<RedbeamsUpgradeState, RedbeamsUpgradeEvent>().defaultFailureEvent(REDBEAMS_UPGRADE_FAILED_EVENT)
-                    .from(INIT_STATE).to(BACKUP_DATABASE_SERVER_STATE).event(REDBEAMS_START_UPGRADE_EVENT).defaultFailureEvent()
-                    .from(BACKUP_DATABASE_SERVER_STATE).to(UPGRADE_DATABASE_SERVER_STATE).event(BACKUP_DATABASE_SERVER_FINISHED_EVENT).defaultFailureEvent()
-                    .from(UPGRADE_DATABASE_SERVER_STATE).to(RESTORE_DATABASE_SERVER_STATE).event(UPGRADE_DATABASE_SERVER_FINISHED_EVENT).defaultFailureEvent()
-                    .from(RESTORE_DATABASE_SERVER_STATE).to(REDBEAMS_UPGRADE_FINISHED_STATE).event(RESTORE_DATABASE_SERVER_FINISHED_EVENT).defaultFailureEvent()
+                    .from(INIT_STATE).to(UPGRADE_DATABASE_SERVER_STATE).event(REDBEAMS_START_UPGRADE_EVENT).defaultFailureEvent()
+                    .from(UPGRADE_DATABASE_SERVER_STATE).to(REDBEAMS_UPGRADE_FINISHED_STATE).event(UPGRADE_DATABASE_SERVER_FINISHED_EVENT).defaultFailureEvent()
```

Comment:
this seems to be a non backward compatible change. what would happen to those upgrades which are already in these state during an upgrade?